src/test/java/org/broadinstitute/hellbender/utils/read/markduplicates/AbstractMarkDuplicatesCommandLineProgramTest.java

```diff
@@ -20,16 +20,22 @@
 
     protected abstract AbstractMarkDuplicatesTester getTester();
 
+  // ELIGIBLE_QUALITY is the minimum quality considered by the dataflow version
+  // for marking a specific pair as best. We use it to ensure that the expected
+  // pair wins the tie so that the tie is not broken randomly.
     protected final static int DEFAULT_BASE_QUALITY = 10;
+    protected final static int ELIGIBLE_QUALITY = 15;
```

Comment:
Sure. This is what is set as MIN_QUAL in the code, and it's the minimum quality that is even considered for scoring. Happy to switch it, if you think BETTER_BASE_QUALITY is still clearer.